react-charts/line-chart/step1.md

```diff
@@ -1,10 +1,21 @@
-PatternFly React charts are based on the [Victory chart](https://formidable.com/open-source/victory/docs/victory-chart/) library, along with additional functionality, custom components, and theming for PatternFly. 
-This provides a collection of React based components you can use to build PatternFly patterns with consistent markup, styling, and behavior. 
+Begin by adding a CSS selector to define the chart size.
 
-PatternFly React charts are composed using separate components, so features are more easily customizable. 
-In this course, we're going to build a PatternFly line chart together - starting with a simple chart, adding multiple datasets, tooltips, axis labels, a legend, and concluding by changing the theme color.
-You'll learn how to use PatternFly React chart components together to build a consistent user experience.
+Katacoda is setting up a new React application. Begin coding once the server starts and "Welcome to PatternFly" is visible on the lower pane.
 
-The image below is what we'll end up with when we are done.
+1) <strong>Navigate to the `src` folder and open `src/app.css`{{open}}</strong>
+
+2) <strong>Add CSS to define the overall chart size.</strong>
+
+Copy the following code into the app.css file, replacing all of the content.

```

Comment:
same comment as the other modules about back-ticks for any file name or piece of code